src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.cpp

```diff
@@ -158,7 +160,7 @@ void SolidMechanicsLagrangianFEM::registerDataOnMesh( Group & meshBodies )
     {
       setConstitutiveNamesCallSuper( subRegion );
 
-      subRegion.registerField< solidMechanics::strain >( getName() ).reference().resizeDimension< 1 >( 6 );
+      subRegion.registerField< solidMechanics::strain >( getName() ).setDimLabels( 1, voightLabels ).reference().resizeDimension< 1 >( 6 );
```

Comment:
what about stress?

Reply:
For stress we also specify our own labels [here](https://github.com/GEOS-DEV/GEOS/blob/c225416b8ca7efcc08e13503f96dbc08e3c925f3/src/coreComponents/constitutive/solid/SolidBase.cpp#L37)